test/token/ERC20/extensions/ERC20FlashMint.test.js

```diff
@@ -92,5 +92,61 @@ contract('ERC20FlashMint', function (accounts) {
       // _mint overflow reverts using a panic code. No reason string.
       await expectRevert.unspecified(this.token.flashLoan(receiver.address, this.token.address, MAX_UINT256, data));
     });
+
+    it('custom flash fee', async function () {
```

Comment:
There is a lot of duplicated code between this test and the next. Can you try to simplify it? Consider using a `describe` block with a `beforeEach`, and/or defining a helper function to do some of the shared checks.